apps/services/user-notification/src/app/modules/notifications/notificationsWorker.service.ts

```diff
@@ -212,7 +212,11 @@ export class NotificationsWorkerService implements OnApplicationBootstrap {
 
     const formattedTemplate = this.notificationsService.formatArguments(
```

Comment:
Imo should shallow copy inside this function instead. It already returns the formatted template and it's an antipattern to mutate function arguments. 🤷‍♂️